commands/files/serialfile.go

```diff
@@ -26,13 +27,17 @@ type serialFile struct {
 }
 
 func NewSerialFile(name, path string, stat os.FileInfo) (File, error) {
-	if stat.Mode()&os.ModeSymlink != 0 {
+	switch mode := stat.Mode(); {
+	case mode.IsDir() || mode.IsRegular():
```

Comment:
I would also prefer and explicit break here. Those of us with C backgrounds may get nervous looking at this.